chef/cookbooks/neutron/templates/default/dnsmasq-neutron.conf.erb

```diff
@@ -0,0 +1,3 @@
+# this file is managed by chef to give extra options to dnsmasq
+dhcp-option-force=26,<%= @mtu.to_s %>
```

Comment:
> I thought, it might be used later for other cases, e.g. let user give a mtu
Even in that case we probably only want to set that option when using a non-default value.
So IMO we should add an ```only_if``` condition to the ```template /etc/neutron/dnsmasq-neutron.conf``` and add a flag to the ```dhcp_agent.ini``` to only include that file when needed.